src/components/Auth/AuthDetails.js

```diff
@@ -43,6 +49,8 @@ function AuthDetails() {
         })
       );
       await updateProfileFields({ variables: { input: userProfile } });
+      await completeRegister(user.id);
```

Comment:
I don't think `user` is defined here. But you can just call `completeRegister` with out any arguments and the backend checks for the current user and sets them as TRUE.

```suggestion
      await completeRegister();
```